hepcrawl/spiders/edp_spider.py

```diff
@@ -0,0 +1,430 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of hepcrawl.
+# Copyright (C) 2016 CERN.
+#
+# hepcrawl is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Spider for EDP Sciences."""
+
+from __future__ import absolute_import, print_function
+
+import os
+import urlparse
+
+import tarfile
```

Comment:
Usually the imports are ordered in three blocks, stdlib, third parties, and intra-package, in this case, tarfile and future are stdlib. Not a big deal though.